process/smartContract/scQueryService_test.go

```diff
@@ -611,17 +611,17 @@ func TestSCQueryService_RecreateTrie(t *testing.T) {
 		argsNewSCQuery.BlockChainHook = &testscommon.BlockChainHookStub{
 			GetAccountsAdapterCalled: func() state.AccountsAdapter {
 				return &stateMocks.AccountsStub{
-					RecreateTrieCalled: func(rootHash []byte) error {
+					RecreateTrieCalled: func(options common.RootHashHolder) error {
+						if options.GetEpoch().HasValue {
+							recreateTrieWasCalled = false
+							recreateTrieFromEpochWasCalled = true
```

Comment:
After the refactor, a slightly better name would be `recreateTrieWasCalledWithEpoch`.